src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_validators.py

```diff
@@ -53,7 +57,7 @@ def validate_client_parameters(namespace):
     if not n.account_key:
         n.account_key = az_config.get('storage', 'key', None)
     if not n.sas_token:
-        n.sas_token = az_config.get('storage', 'sas_token', None)
+        n.sas_token = os.environ.get('AZURE_STORAGE_SAS_TOKEN')
```

Comment:
That would be a case of when git good goes bad.

---
In reply to: [78047355](https://github.com/Azure/azure-cli/pull/843#discussion_r78047355) [](ancestors = 78047355)